CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Changelog
 
+## 0.1.2
```

Comment:
Let's add:

```
## Highlights

The Ruff formatter is an extremely fast, Black-compatible Python formatter, out now as a Beta release. Try it today with `ruff format`.
```

Wdyt? And then we can add a link to the blog post once it's up.

Reply:
Added with some edits